r-package/grf/R/local_linear_forest.R

```diff
@@ -5,6 +5,16 @@
 #'
 #' @param X The covariates used in the regression.
 #' @param Y The outcome.
+#' @param ll.splits Optional choice to make forest splits based on ridge residuals as opposed to standard
```

Comment:
Maybe we could rename this to `enable.ll.split`? I had a hard time grokking what `ll.splits` meant at first.

Also should we mark LL splitting as experimental? We could let it 'bake' for a bit in case we want to make modifications.

Reply:
Yes! I'll add a note in the parameter description; is there a more official way to mark it as experimental?